lib/src/utils.dart

```diff
@@ -76,6 +76,47 @@ FileSpan extendForward(FileSpan span, String text) {
   return span.file.span(span.start.offset, end + text.length);
 }
 
+/// Returns the next location after [import] that it would be safe to insert
+/// a `@use` or `@forward` rule.
+///
+/// This is generally the start of the next line, but may vary if there's any
+/// non-whitespace, non-comment code on the same line.
+FileLocation afterImport(ImportRule import, {bool shouldHaveSemicolon = true}) {
```

Comment:
You might be able to simplify this by comparing the end location of `import` with the start location of the next statement in `import.parent.children` (ignoring comments). If they're on different lines, you can just choose the end-of-line location; if they're not, you can choose the beginning of the next statement instead.

Reply:
I considered that, but `SassNode`s don't actually have a `parent` property (only `ModifiableCssNode`s do as far as I can tell).